.github/dependabot.yml

```diff
@@ -9,5 +9,7 @@ updates:
   labels:
   - dependabot
   versioning-strategy: increase-if-necessary
+  allow:
+    - dependency-type: "direct"
```

Comment:
What about security patches in transient-dependencies?

Reply:
I think Snyk takes care of those, right?

Reply:
Yes that is correct!

Reply:
Plus when we run an npm install in this repo anyway we update the package-lock file so any patches get picked up with this